src/yajra/Datatables/Request.php

```diff
@@ -134,7 +134,8 @@ public function isColumnSearchable($i, $column_search = true)
     {
         $columns = $this->get('columns');
         if ($column_search) {
-            return $columns[$i]['searchable'] == 'true' && $columns[$i]['search']['value'] != '' && ! empty($columns[$i]['name']);
+            return $columns[$i]['searchable'] == 'true' && $columns[$i]['search']['value'] != '';
```

Comment:
Can you please elaborate the need to remove the check for name index? We need the name index for searching and ordering of queries so it's important to check it before hand. Thanks!